docs/performance.md

```diff
@@ -1,194 +1,88 @@
 # Tessera Storage Performance
 
-This document describes the performance of each Trillian Tessera storage implementation under different conditions.
+Tessera is designed to scale up and down to meet the needs of most currently envisioned workloads in a cost-effective manner.
```

Comment:
Do you mean scale out and in instead of up and down unless only one instance is allowed?

Reply:
I would just say that it is designed to be scalable. Whether scaling is done automatically and whether that's horizontally/vertically is outside the scope of this introductory sentence, IMO.